support-frontend/assets/helpers/redux/commonState/selectors.ts

```diff
@@ -55,3 +55,16 @@ export function isUserInAbVariant(abTestName: string, variantName: string) {
 		return participations[abTestName] === variantName;
 	};
 }
+
+export function isUserInAnyAbVariant(
```

Comment:
I believe we agreed this `isUserInAnyAbVariant` function can be removed as it's not used anywhere in the code?